README.md

```diff
@@ -234,6 +235,33 @@ PYTHONPATH=python/gradbench/:$PYTHONPATH ./gradbench repo run --eval "$ python3
 GradBench if you wish.](cpp#from-the-command-line) This does require
 you to first extract the raw input from a `gradbench` log file.
 
+### Multithreading
+
+By default, tools use only a single thread. You can ask them to use multiple
+(CPU) threads, if possible, by passing the option `--multithread`. If you use
+`gradbench repo tool`, then remember to pass `--` before any tool-specific
+options. Example:
+
+```sh
+./gradbench repo tool manual -- --multithread
+```
```

Comment:
I think using `repo tool` here is out of place now that #522 has edited all this documentation to use `repo run` instead. Can you update this section to use `repo run` as well?